document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:
+1) Important item

```

Comment:
This is a test

---
In reply to: [54169441](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r54169441) [](ancestors = 54169441,54165983,54165527)